front/src/Components/Chat/ChatUser.svelte

```diff
@@ -0,0 +1,36 @@
+<script>
```

Comment:
@cesar-cardinale this file is used in `front/src` or already moved in `front/chat/src`?